src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -192,16 +192,16 @@ public LocalMovie GetLocalMovie(string filename, ParsedMovieInfo minimalInfo, Mo
             };
         }
 
-        public ParsedMovieInfo ParseMinimalMovieInfo(string file)
+        public ParsedMovieInfo ParseMinimalMovieInfo(string file, bool isDir = false)
         {
-            return Parser.ParseMovieTitle(file, _config.ParsingLeniency > 0);
+            return Parser.ParseMovieTitle(file, _config.ParsingLeniency > 0, isDir);
         }
 
         public ParsedMovieInfo ParseMinimalPathMovieInfo(string path)
         {
             var fileInfo = new FileInfo(path);
 
-            var result = ParseMinimalMovieInfo(fileInfo.Name);
+            var result = ParseMinimalMovieInfo(fileInfo.Name, true);
```

Comment:
Have you tried yet whether this actually works? Because it doesn't really make sense to put it here, since that is actually just the filename. (I know that's where it originally was, but no idea why that worked either lol)

Reply:
Yes, tested it and it worked with both these folder names:

- `[2018] The Most Epic Movie Ever`
- `The Most Epic Movie Ever (2018)`

I don't know how some things work either so this was a bit of trial and error, hopefully it doesn't break anything else :)